packages/playground/windows/NuGet.Config

```diff
@@ -3,11 +3,4 @@
   <config>
     <add key="repositoryPath" value="packages" />
   </config>
-  <packageSources>
-    <clear />
-    <add key="Nuget.org" value="https://api.nuget.org/v3/index.json" />
-  </packageSources>
-  <disabledPackageSources>
```

Comment:
This is interesting as this was required to be injected? Is a new Nuget Version 'better' at not including random nuget.config files from all over the place and affecting determinisitc builds?

Reply:
Right, except nuget merges all the config files up the folder hierarchy. So by only specifying the sources in the root config, there's only one file for the task to complain about.